routing/mock_graph_test.go

```diff
@@ -13,7 +14,12 @@ import (
 
 // createPubkey return a new test pubkey.
 func createPubkey(id byte) route.Vertex {
-	pubkey := route.Vertex{id}
+	_, secpPub := btcec.PrivKeyFromBytes([]byte{id})
```

Comment:
We needed to change this because the test required the private key to be a real one. Just mocking with fixed values would give us an error parsing the key. This change makes it so the parsing works because we use real generated values. Do you think this makes sense?